### PR TITLE
Response format

### PR DESCRIPTION
**Pull request by matthewcharles27**
added response format section to ocs-api-reference.md. removed specifics from other 2 docs, and added a link to details.

### CODE REVIEW

File: content/api-reference/ocs-api-reference.md [hunk 1]
```diff
@@ -34,12 +34,23 @@ You must acquire an account before interacting with AVEVA Data Hub. To request a
 
 For information on the graphical interface for using the REST API within AVEVA Data Hub, see [API console](xref:apiConsole).
 
+### Response format
```
Comment (osisoft-mbishop) [inline content/api-reference/ocs-api-reference.md:37]: Bump the heading up a level:
```suggestion
## Response format
```

File: content/developer-guide/data-views-dev/get-data-view-data/quick-start-get-data-view-data.md [hunk 2]
```diff
@@ -115,9 +115,7 @@ The view data is available in several formats. A format can be specified using t
 
 #### Response format
 
-By default, the value of the header `Accept-Verbosity` is `verbose` for the response format. Verbose responses include all values in the returned JSON payload, including defaults. When set to `non-verbose`, properties with null values are omitted from the response to reduce bandwidth. This behavior is only applicable to the JSON format, as all other formats represent a table which cannot have omitted values.
-
-Note that the `Accept-Verbosity` header for SDS differs as its default is set to `non-verbose`.
+By default, the value of the header `Accept-Verbosity` is `verbose` for the response format. For more information on response formats, see [AVEVA Data Hub API reference](xref:osisoftCloudServices).
```
Comment (osisoft-mbishop) [inline content/developer-guide/data-views-dev/get-data-view-data/quick-start-get-data-view-data.md:118]: Update this lead-in text to match the lead in text for the next topic. This content is weird--it's talking about options that aren't relevant until you set a response format

**Review by osisoft-mbishop — APPROVED** [review]

**Comment by osisoft-mbishop** [comment]
@derekendres, can you review this doc PR for moving the accept verbosity header? Please let @matthewcharles27 if any updates are needed.

Thanks!

File: content/api-reference/ocs-api-reference.md [hunk 3]
```diff
@@ -34,12 +34,23 @@ You must acquire an account before interacting with AVEVA Data Hub. To request a
...
+
+Supported response formats include JSON, verbose JSON, and SDS.
+
+By default, the value of the header `Accept-Verbosity` is `verbose` for the response format. Verbose JSON responses include all values in the returned JSON payload, including defaults. To specify verbose JSON return, add the header ``Accept-Verbosity`` with a value of ``verbose`` to the request.
```
[the remark this answers is not in the document]
Comment (derekendres) [reply content/api-reference/ocs-api-reference.md:41]: yes looks good

**Review by derekendres — APPROVED** [review]

**Review by osisoft-mbishop — APPROVED** [review]

**Pull request merged**